source/appendices/e-audit-requirements-and-tests/05-book-usage-tests.rst

```diff
@@ -0,0 +1,98 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.5 Audit Tests for Investigations and Requests for Books and Reference_Works
+-----------------------------------------------------------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for any platform delivering Book and/or Reference_Work content. For report providers operating multiple platforms including one or more with Book and/or Reference_Work content, the audit scope as defined in :numref:`audit` MUST include one of these platforms and be subject to the audit tests outlined here. Note that these tests refer to Data_Types Book and Book_Segment for ease of reading. In all cases, Book includes Data_Types Book and Reference_Work, while Book_Segment includes Data_Types Book_Segment and Reference_Item.
+
+These tests apply to the Platform Report, Database Report, Title Report and Item Report. The PR_P1, DR_D1, TR_B1 and TR_B3 Standard Views of the COUNTER Reports will be deemed COUNTER-compliant if the metrics and Data_Types match those in the relevant COUNTER Report, with the appropriate aggregation. 
```

Comment:
Maybe it would be worth mentioning here and in E.6 that the counts specified are for TR and IR, and that they have to be aggregated appropriately for DR and PR (in the cases with two or three Access_Types)?